packages/manager/src/features/Betas/index.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { SuspenseLoader } from 'src/components/SuspenseLoader';
+
+const BetasLanding = React.lazy(() => import('./BetasLanding'));
+
+const BetaSignup = React.lazy(() => import('./BetaSignup'));
+
+const BetaRoutes = () => (
+  <React.Suspense fallback={<SuspenseLoader />}>
+    <Switch>
+      <Route component={BetaSignup} path="/betas/betaId:signup" />
```

Comment:
we'll need to change this to

```suggestion
      <Route component={BetaSignup} path="/betas/:betaId/signup" />
```

for the routing to work properly